src/altair/webhook.py

```diff
@@ -44,6 +44,11 @@ async def process_update(
     log.info("WEBHOOK")
     req = await request.json()
 
+    if req["message"]["chat"]["id"] != Config.APPROVED_USER:
+        # Ignore all messages that aren't from Matt, return stub response.
+        # We want to tell Telegram the payload is OK, just not the right user.
+        return Response(status_code=HTTPStatus.OK)
```

Comment:
I do realise it's not clear from reading the code, but I think this is better handled in the bot as you can send a friendly response like 🖕

Reply:
I would not get the bot to respond, personally. My Git update bot, for example, or the Cloud Gaming bot we collab'd on, did not respond to 'non-owner' messages.

Think of it as port scanning.

Reply:
That's fair. I don't know for certain but I think PTB validates responses, which should handle scans.

Reply:
Yeah, I'm just thinking of this from a cost and security standpoint. Better to remain silent and be thought a fool, than to speak, and to remove all doubt.